src/generic_ui/scripts/ui.ts

```diff
@@ -117,6 +117,7 @@ module UI {
     USER,
     NETWORKS,
     SETTINGS,
+    BROWSER
```

Comment:
Can we change this to something more specific like APP_MISSING or at least BROWSER_ERROR?  I know APP_MISSING sounds Chrome specific but BROWSER is a little too vague and I don't think I'll remember what this means in a week.